android/app/src/main/java/org/pathcheck/covidsafepaths/storage/Location.kt

```diff
@@ -0,0 +1,91 @@
+package org.pathcheck.covidsafepaths.storage
+
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.ReadableType
+import com.marianhello.bgloc.data.BackgroundLocation
+import io.realm.RealmObject
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.bridge.WritableNativeMap
+import io.realm.annotations.PrimaryKey
+import java.lang.Exception
+
+/*
+  Realm requires a no-op constructor. Need to use var and fill will default value
+ */
+open class Location(
```

Comment:
so this has to be declared as open class?

Reply:
Yes, sounds like realm does something to inherit from this class so we need to leave it open.